software_subprojects.md

```diff
@@ -0,0 +1,31 @@
+# Software Subprojects of Project Jupyter
+
+Software design and development in Project Jupyter is organized into a set of Software Subprojects. These software Subprojects often map onto GitHub repositories/organizations, but that is not strictly required. The lifecycle of Software Subprojects is [described in detail here](newsubprojects.md).
+
+Since Project Jupyter promotes open technical standards and interoperability among components, there is a rich ecosystem of tools developed by third parties that are not under the umbrella of Project Jupyter’s Governance or the Software Steering Council. We strive to maintain open communication with this community, however,this document focuses only on official Jupyter Subprojects.
```

Comment:
I would remove this paragraph in its entirety, it's an aspiration not a governance element.

Reply:
Good point, and shorter is better (quoting @afshin :) - nuking.